app/models/event.rb

```diff
@@ -29,16 +31,24 @@ class Event < ApplicationRecord
 
   scope :active, -> { where('deadline >= ?', Date.current) }
 
-  validates :name, :discipline, :game_mode, presence: true
-  validates :name, :discipline, :game_mode, :player_type, presence: true
-  validates :deadline, :startdate, :enddate, presence: true
+  validates :name, :discipline,  presence: true
+  validates :deadline, :startdate, :game_mode, :player_type, :enddate,
```

Comment:
This would make more sense in the Tournament and League subclasses (where those fields are actually required fields).